api/src/main/java/openconsensus/trace/Event.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019, OpenConsensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package openconsensus.trace;
+
+import java.util.Map;
+import javax.annotation.concurrent.Immutable;
+import openconsensus.trace.data.AttributeValue;
+
+/**
+ * A text annotation with a set of attributes.
+ *
+ * @since 0.1.0
+ */
+@Immutable
+public abstract class Event {
```

Comment:
API should allow to create an Event. We don't have this option any longer, do we?

Reply:
User can add events via the Span.event api which supports all the variants. Do we still need an event implementation here?

Reply:
How do you create event for spandata?

Reply:
Done.

Reply:
See my comment. Any reason to create public type, not simply return it from the static method on `Event`?

Reply:
Done. Let's address the interface change separately.